translations/index.ts

```diff
@@ -52,5 +52,6 @@ export function translate (path: string, text: string): string {
   if (!init) {
     initI18n()
   }
-  return i18n.translate(`${path}.${text}`)
+  const translation = i18n.translate(`${path}.${text}`)
+  return translation != null && translation !== '' ? translation : text
```

Comment:
I prefer if condition than ternary if your has more than 1 condition, in this case your ternary has 2 conditions.

`if ()` allows for better code readability as you can clearly see where the code condition exits or continue.

Reply:
updated